example/cumulus-tf/main.tf

```diff
@@ -109,6 +109,7 @@ module "cumulus" {
 
   archive_api_port = var.archive_api_port
   private_archive_api_gateway = var.private_archive_api_gateway
+  api_gateway_stage = var.api_gateway_stage
```

Comment:
Does this need to be added to the template?   That question can probably be addressed by the task on the docs TODO as well.

Reply:
It probably makes sense to, since I assume folks don't want their prod stages to be named `dev`.